### ISSUE TITLE
dataframe for multi label multi class classification

### ISSUE THREAD

**Issue by buffyhridoy** [keras-team/keras#15670]
This is the colab link for the problem I am facing
https://colab.research.google.com/drive/1lsbXtqkA5e_GJW0mO05Jw-wc6_Vbf9QL?usp=sharing#scrollTo=BodcE3pjvF20

**Comment by old-school-kid** [comment]
@buffyhridoy can you be a little elaborative with the issue. The notebook doesnt even have keras imported.

**Comment by chunduriv** [comment]
@buffyhridoy, In order to expedite the trouble-shooting process, please elaborate your issue on keras. Thanks!
